ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -96,7 +96,7 @@ function NetworkGraphPage() {
     );
 
     const [isLoading, setIsLoading] = useState(false);
-    const [timeWindow, setTimeWindow] = useState<(typeof timeWindows)[number]>(timeWindows[0]);
+    const [timeWindow, setTimeWindow] = useState<TimeWindow>(timeWindows[0]);
```

Comment:
Two questions here:
1. Was this value an orphan? That is, set in `TimeWindowSelector` element, but not otherwise used?
2. What are pro and con to rename its props? Simple minded search failed on first attempt.
    * Replace `activeTimeWindow={timeWindow}` with `timeWindow={timeWindow}`
    * Replace `setActiveTimeWindow={setTimeWindow}` with `setTimeWindow={setTimeWindow}`

Reply:
Aha, another name. Two’s company, three’s a crowd. All the more reason to reduce from 3 to 2.

```js
const timestampToUse = timeWindowToDate(timeWindow);
```

Reply:
I think you answered question 1 on your own and I'd agree that renaming props to stay consistent makes sense. I'll go ahead and update.